packages/wonder-blocks-theming/package.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "@khanacademy/wonder-blocks-theming",
+    "version": "0.0.1",
+    "publishConfig": {
+        "access": "public"
+    },
+    "description": "",
```

Comment:
**nit:** Add a simple description, like:

`Provide theming support for Wonder Blocks components"